Mrsj/addpages

matthewrsj opened the pull request:
* Migrated content to new site (ref #6)
* Formatted regular pages and blog posts (ref #5)

There is still some formatting and other issues that need to be worked on to mirror the original site, including:
* Forms have still not been completed
* Menu is still hard-coded
* CSS is still a mess

matthewrsj commented:
@Kennric @ramereth @jordane can you review this when you get a chance?

ramereth commented on content/about/about_people.rst:
```diff
@@ -0,0 +1,371 @@
+    :alt: Lance Albertson
+
+| **Director | Cat Herder**
+| `Lance <lancealbertson.com>`_ became OSL director in early 2013. He has managed all of the hosting activities that the OSL provides for more than 160 high-profile open source projects since joining the lab as lead systems administrator and architect in 2007. Lance’s involvement in the open source community began in 2003, when he became a developer and package maintainer with `Gentoo Linux <gent ...
```
Manually line wrap this and all other occurrences.

ramereth commented on content/about/about_people.rst:
```diff
@@ -0,0 +1,371 @@
+`Jeff Borcean`_
+---------------
+
+.. image:: /theme/img/Jeffcropped12-15.jpg
```
It would be good to standardize the naming of these image files.

ramereth commented:
@matthewrsj Also, in general, I like putting all the hyper links below each paragraph.. So an example would be going from something like:

```
`Foo <example.org>_ blah blah blah. `Bar <example.com>`_ blah blah blah.

# to this
`Foo`_ blah blah blah. `Bar`_ blah blah blah.

.. _Foo: example.org
.. _Bar: example.com
```

matthewrsj commented:
@ramereth OK, I will work on all this.

matthewrsj commented:
@ramereth I cleaned up the line lengths and moved the  hyperlinks. I also found some content that was missing/duplicated.

Kennric commented on content/about/summary/students.rst:
```diff
@@ -0,0 +1,52 @@
+Students
+=========
```
Too many =

Kennric commented on content/about/summary/students.rst:
```diff
@@ -0,0 +1,52 @@
+working in developer and system administrator positions.
+
+`Student Stories`_
+--------------------------------------
```
Too many -

Kennric commented on content/about/summary/students.rst:
```diff
@@ -0,0 +1,52 @@
+.. _student stories: /students/stories
+
+`Open Source on Campus`_
+------------------------------------------------
```
Too many -

matthewrsj replied:
OK, I can fix all these. I thought it was OK if it was at least as long as the text.

Kennric commented on content/blog/posts/open-source-bridge-promotes-community.rst:
```diff
@@ -1,52 +1,89 @@
 Open Source Bridge Promotes Community
 #####################################
```
Lets be consistent, either all ===== or all ###### for top level headers (i think === is more standard)

Kennric commented on content/blog/posts/osl-staff-mentors-international-students-through-google-summer-of-code.rst:
```diff
@@ -1,51 +1,90 @@
+Media contact: Kayla Harr, harrk@osuosl.org To learn more about the students who
+gain real-world experience through the Open Source Lab, visit our `staff`_ page.
+More information about Google Summer of Code and how to participate can be found
+`here`_.
```
Let's not use generics like 'here' for named link references - I think "can be found at the `OSL GSOC page`_ or something like that.

Kennric commented on content/services/powerlinux/services_powerdev_request_hosting.rst:
```diff
@@ -0,0 +1,45 @@
+PowerLinux / OpenPOWER Request Form
+====================================
```
Too many =

Kennric commented on content/services/services.rst:
```diff
@@ -0,0 +1,81 @@
+
+
+`Hosting Services`_
+--------------------
```
Too many -

Kennric commented on content/services/services.rst:
```diff
@@ -0,0 +1,81 @@
+
+
+`Supercell`_
+-------------
```
Too many -

Kennric commented on content/services/services_development.rst:
```diff
@@ -0,0 +1,115 @@
+Development
+============
```
Too many =

Kennric commented on content/services/services_development.rst:
```diff
@@ -0,0 +1,115 @@
+
+
+`Ganeti Web Manager`_
+----------------------
```
Too many -

Kennric commented on content/services/services_development.rst:
```diff
@@ -0,0 +1,115 @@
+
+
+`Protein Geometry Database`_
+-----------------------------
```
Too many -

Kennric commented on content/services/services_hosting.rst:
```diff
@@ -0,0 +1,51 @@
+
+
+Managed and Unmanaged Hosting
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Consistency - instead of ~

Kennric commented on content/services/services_hosting.rst:
```diff
@@ -0,0 +1,51 @@
+
+
+To Request Hosting
+~~~~~~~~~~~~~~~~~~
```
Consistency - instead of ~

Kennric commented on content/services/supercell/services_supercell.rst:
```diff
@@ -0,0 +1,147 @@
+
+
+What is Supercell?
+~~~~~~~~~~~~~~~~~~
```
Consistency - instead of ~

Kennric commented on content/services/supercell/services_supercell.rst:
```diff
@@ -0,0 +1,147 @@
+
+
+Which guest operating systems does Supercell support?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Consistency - instead of ~

Kennric commented on content/services/supercell/services_supercell.rst:
```diff
@@ -0,0 +1,147 @@
+
+
+Do you have plans to support other guest operating systems?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Consistency - instead of ~

Kennric commented on content/services/supercell/services_supercell.rst:
```diff
@@ -0,0 +1,147 @@
+
+
+What about testing on OSX?
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Consistency - instead of ~

Kennric commented on content/services/supercell/services_supercell.rst:
```diff
@@ -0,0 +1,147 @@
+
+
+Do you have plans to support architectures beyond x86?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Consistency - instead of ~

Kennric commented on content/services/supercell/services_supercell.rst:
```diff
@@ -0,0 +1,147 @@
+
+
+What kind of hardware is used for the Supercell cluster?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Consistency - instead of ~

Kennric commented on content/services/supercell/services_supercell.rst:
```diff
@@ -0,0 +1,147 @@
+
+
+Where can I learn more about the Supercell architecture?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Consistency - instead of ~

Kennric commented on content/services/supercell/services_supercell.rst:
```diff
@@ -0,0 +1,147 @@
+
+
+When can I use Supercell?
+~~~~~~~~~~~~~~~~~~~~~~~~~
```
Consistency - instead of ~

Kennric commented on content/services/supercell/services_supercell.rst:
```diff
@@ -0,0 +1,147 @@
+
+
+How much will it cost for open source projects to use Supercell?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Consistency - instead of ~

Kennric commented on content/services/supercell/services_supercell_sponsors.rst:
```diff
@@ -0,0 +1,39 @@
+Sponsors
+=========
```
Too many =

Kennric commented on content/services/supercell/services_supercell_sponsors.rst:
```diff
@@ -0,0 +1,39 @@
+
+
+`Facebook`_
+------------
```
Too many -

Kennric commented:
Content looks good, comments on headers and misc added.

matthewrsj commented:
@Kennric standardized headers and fixed generic reference. In services_development.rst I changed the header underlines so sub-section is '-' and sub-sub-section is '~' to remain consistent with the rest of the site.
